### PR TITLE
[api] fixes #1995 - Measure start time for Skycoin service.

### LINKED ISSUE #1995: Add `started_at` to /health
See: https://github.com/skycoin/skycoin/pull/1976#discussion_r228721930

### PR DESCRIPTION
**Pull request by stdevStark**
Fixes #1995 

Changes:
- `started_at` exports through REST API Unix timestamp for the moment since when uptime is measured
- `make generate-golden-files`

Does this change need to mentioned in CHANGELOG.md?
yes

### CODE REVIEW

**Comment by therealssj** [comment]
do we need to keep `Uptime` if we have multiple `StartedAt` times
wont it become ambiguous as to what the difference is between `Uptime` and the other 3 `StartedAt` times

**Comment by therealssj** [comment]
also the tests need to be updated.
to update the golden files for integration tests run the scripts in `ci-scripts` folder with `-u` to update the files. 
`-c -u` for updating the csrf enabled golden files

**Comment by stdevStark** [comment]
@therealssj it depends on the usage . Keeping them all is no big deal. I added all three timings so that it'd be possible to determine how long initialization was taking in practice . If you prefer to delete `Uptime` it's no big deal , can be easily removed .

**Comment by therealssj** [comment]
@stdevStark its going to cause confusion since its not clear as to what `Uptime` is when we have 3 other timestamps as well
imo it should be removed.
@olemis thoughts?

**Comment by olemis** [comment]
@stdevStark should be only one based on the value already available at `gw.v.StartedAt`

**Comment by therealssj** [comment]
oh if we decide to keep only one then close this PR. 
I have already made the changes using `gw.v.StartedAt` was just about to PR when you opened yours.

**Comment by stdevStark** [comment]
@olemis , fixed considering your suggestion . @therealssj updated golden files .

**Comment by therealssj** [comment]
@stdevStark 
```
./ci-scripts/integration-test-stable.sh -u
./ci-scripts/integration-test-stable.sh -c -u
./ci-scripts/integration-test-stable.sh -d -u
./ci-scripts/integration-test-stable.sh -c -d -u
```
these are the 4 commands that u need to run to update all the files
maybe make a `Makefile` command like 

```
update-golden-files: ## Run integration tests in update mode
	./ci-scripts/integration-test-stable.sh -u >/dev/null 2>&1
	./ci-scripts/integration-test-stable.sh -c -u >/dev/null 2>&1
	./ci-scripts/integration-test-stable.sh -d -u >/dev/null 2>&1
	./ci-scripts/integration-test-stable.sh -c -d -u >/dev/null 2>&1
```

**Comment by stdevStark** [comment]
I did two . Did not know `-d` . Thanks !

**Comment by stdevStark** [comment]
Problem . In integration tests , because `started_at` is absolute , value can change between tests . Nonetheless golden files include a value . @olemis @therealssj  how to fix that ?

**Comment by therealssj** [comment]
update changelog as well

**Comment by olemis** [comment]
@stdevStark golden files still need to be updated after 593eca0 .

**Comment by stdevStark** [comment]
@olemis @therealssj thanks => https://travis-ci.org/skycoin/skycoin/builds/456375777

File: Makefile [hunk 1]
```diff
@@ -255,6 +256,12 @@ generate-mocks: ## Regenerate test interface mocks
 	mv ./src/visor/blockdb/mock_unspent_pooler_test.go ./src/visor/mock_unspent_pooler_test.go
 	sed -i "" -e 's/package blockdb/package visor/g' ./src/visor/mock_unspent_pooler_test.go
 
+generate-golden-files: ## Run integration tests in update mode
```
Comment (gz-c) [inline Makefile]: rename `update-golden-files`

File: Makefile [hunk 2]
```diff
@@ -7,7 +7,8 @@
 .PHONY: integration-test-disable-gui integration-test-disable-gui
 .PHONY: integration-test-db-no-unconfirmed integration-test-auth
 .PHONY: install-linters format release clean-release clean-coverage
-.PHONY: install-deps-ui build-ui help newcoin generate-mocks merge-coverage
+.PHONY: install-deps-ui build-ui help newcoins merge-coverage
+.PHONY: generate-mocks generate-golden-file
```
Comment (gz-c) [inline Makefile]: `update-golden-files`

Comment (stdevStark) [reply Makefile]: completed

**Comment by stdevStark** [comment]
@gz-c uptime is there , not deleted .

File: Makefile [hunk 3]
```diff
@@ -7,7 +7,8 @@
 .PHONY: integration-test-disable-gui integration-test-disable-gui
 .PHONY: integration-test-db-no-unconfirmed integration-test-auth
 .PHONY: install-linters format release clean-release clean-coverage
-.PHONY: install-deps-ui build-ui help newcoin generate-mocks merge-coverage
+.PHONY: install-deps-ui build-ui help newcoins merge-coverage
+.PHONY: generate-mocks update-golden-file
```
Comment (gz-c) [inline Makefile]: `update-golden-files`

File: CHANGELOG.md [hunk 4]
```diff
@@ -54,6 +54,7 @@ Make sure to upgrade to v0.25.0 so that your node will continue to connect once
 - `/api/v1/network/disconnect` to disconnect a peer
 - Complete support for `cipher` package in `libskycoin` C API.
 - Add `coin`, `wallet`, `util/droplet` and `util/fee` methods as part of `libskycoin` C API
+- Add `make generate-golden-files` to `Makefile`
```
Comment (gz-c) [inline CHANGELOG.md]: `update-golden-files`

**Pull request merged**